app/api/dao/user.py

```diff
@@ -50,6 +51,11 @@ def create_user(data):
         if 'available_to_mentor' in data:
             user.available_to_mentor = data['available_to_mentor']
 
+        gravatar = Gravatar(email)
+        gravatar_url = gravatar.get_image(size=512)
+
+        user.photo_url = gravatar_url
```

Comment:
IMO, we can put this in a try-catch since we are calling an external API.
WDYT?

Reply:
very good idea indeed, fix incoming🛠

Reply:
Done!